kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -22,15 +32,35 @@ def airflow_commands():
     pass
 
 
+def _load_config(
+    context: KedroContext, pipeline_name: str, patterns: list[str]
+) -> dict[str, Any]:
+    try:
+        config_airflow = context.config_loader.get(*patterns)
```

Comment:
We recently came across this problem with other plugins as well. This syntax does not work for `OmegaConfigLoader` as it does not have a `get()` method. The config should be loaded like a dict - `context.config_loader['airflow']`

Reply:
Do you have a link to one of those other plugins by any chance?

Reply:
This is the [issue I created on `kedro-mlflow`](https://github.com/Galileo-Galilei/kedro-mlflow/issues/430). There is a PR linked to it you can take a look at!

Reply:
Thank you!